src/js/startApp.js

```diff
@@ -1,17 +1,39 @@
-import { Station } from './Station';
-import { SubwayLine } from './SubwayLine';
-import { Subway } from './Subway';
-import { Route } from './Route';
+import { createSubway } from './subway func'
+import { Parser } from './Parser'
 
 let fileReader = new FileReader();
+let parser = new Parser();
 let textInput = document.querySelector('#file-input');
+let currentSubway = null; 
 let subwayData = null;
+const lsKeys = {
```

Comment:
it is worth to create separate constant for this. also it is worth to name it with full name localStorageKeys